doc/cephfs/eviction.rst

```diff
@@ -23,15 +23,15 @@ Automatic client eviction
 
 There are three situations in which a client may be evicted automatically:
```

Comment:
I think dropping this colon would fix the issue.

Reply:
I will edit that :) - thanks for the info ^_^